src/classlibnative/inc/nlsinfo.h

```diff
@@ -101,10 +101,8 @@ class COMNlsInfo {
     //
     static FCDECL1(FC_BOOL_RET, nativeInitCultureData, CultureDataBaseObject *data);
     static FCDECL3(FC_BOOL_RET, nativeGetNumberFormatInfoValues, StringObject* localeNameUNSAFE, NumberFormatInfo* nfi, CLR_BOOL useUserOverride);
-#ifdef FEATURE_USE_LCID
```

Comment:
I am focusing on enabling this for Windows. I am going to port all needed functionality to Linux later. for Linux we don't have to use exact same same code used for Windows (because it is Windows dependent code) and we'll have different implementation. for serialization specifically, what I am enabling here should help the serialization on Windows side because I am bringing the fields used in the desktop serialization. we'll need to fix teh serialization on Linux though which I was planning to look at that when I finish the whole implementation.

---
In reply to: [75938164](https://github.com/dotnet/coreclr/pull/6866#discussion_r75938164) [](ancestors = 75938164)